adagrams/game.py

```diff
@@ -1,13 +1,135 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+def create_pool_list():
+    pool_list = []
+    for letter, number_of_tiles in LETTER_POOL.items():
+        for index in range(0, number_of_tiles):
+            pool_list.append(letter)
+    return pool_list
```

Comment:
Love that you are using helper functions! It will make your code more modular! Something to note about your `for range` loop here: 

 When we are using a for range loop that doesn't utilizes the loop variable, `index`, we can instead use `_` to signal to other developers and ourselves that this loop is just to run our iteration logic **x** amount of times. 
```suggestion
def create_pool_list():
    pool_list = []
    for letter, number_of_tiles in LETTER_POOL.items():
        for _ in range(0, number_of_tiles):
            pool_list.append(letter)
    return pool_list
```